modules/tinymce/CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Table toolbar was visible even if the table was within a noneditable host element. #TINY-9664
 - Quickbar toolbars was shown for noneditable contents in a noneditable root. #TINY-9460
 
+### Changed
+- Update the dropdown style of code sample and template to tinymce custom style. #TINY-9630
```

Comment:
Thinking this might be a bit clearer as to what the change is.
```suggestion
- Update the `codesample` plugin dialog and `template` plugin dialog to use the 'listbox' component to match other dialogs. #TINY-9630
```